service/spi/src/main/java/gov/va/vro/service/spi/model/GeneratePdfPayload.java

```diff
@@ -27,6 +27,8 @@
 public class GeneratePdfPayload {
   @NonNull private String claimSubmissionId;
 
+  @Builder.Default @NotNull private String idType = Claim.V1_ID_TYPE;
```

Comment:
Yes it's used in both V1 and v2 workflows. It is currently set to default and then v2 explicitly sets it.
I don't entirely disagree that this could be set explicitly in v1 but ensuring I get all the places this should be set at this late state is more risky than addressing it later imo.